config.py

```diff
@@ -60,7 +60,7 @@ def main():
             args.pckgidir = '$SCRATCH'
 
     config_data['config']['install_tree'] = args.pckgidir + '/spack-install/' + args.machine.replace('admin-', '')
-    config_data['config']['build_stage'] = [args.pckgidir + '/spack-stages/' + args.machine]
+    config_data['config']['build_stage'] = ['$SCRATCH' + '/spack-stages/' + args.machine]
```

Comment:
what about adding -admin if admin (so creating a variable for stage): path would be:
'$SCRATCH' + '/spack-stages-admin/'

Reply:
we need somehow to separate install and stage. If the issue that args.pckgidir is used somewhere else